asset/scss/_header.scss

```diff
@@ -3,26 +3,39 @@
     padding-top: 2rem;
     padding-bottom: 2rem;
 
-    header {
+    .header-slim {
```

Comment:
My understanding that I want to confirm is that `_header.scss` is omeka specific and did not originate from our style library? I want to make sure I understand that these features are still additive to our upstream theme and not changing it in this file.

Either way, as it was noted we may want to include this feature in our style guide please open an EngX ticket and bring it to the team for discussion. We've also identified some needed changes around skip links and a few other bits recently so we can discuss a few related things at our next team meeting

Reply:
I've opened https://mitlibraries.atlassian.net/browse/ENGX-261 to move this feature up to the style library.

And yes, the `_header.scss` file is Omeka-specific. The upstream style library lands in this theme via the compiled stylesheet, not the individual SCSS modules.